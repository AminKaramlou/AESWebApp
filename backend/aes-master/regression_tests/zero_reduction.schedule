1: A B
2: C D
3:
