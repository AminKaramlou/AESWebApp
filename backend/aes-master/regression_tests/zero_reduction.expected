Schedule is feasible
 -  All jobs are allocated to exactly one machine
Schedule does satisfies user fixed decisions
 -  All jobs satisfy all fixed decisions
Schedule is not efficient because:
 -  Job A can be allocated from machine 1 to 3 to reduce by 0
 -  Job B can be allocated from machine 1 to 3 to reduce by 0
